sdk/python/setup.sh

```diff
@@ -5,7 +5,7 @@ pip install azure-ai-ml
 # </az_ml_sdk_install>
 
 # <mldesigner_install>
-pip install mldesigner
+pip install mldesigner==0.1.0b7
```

Comment:
revert this ?

Reply:
Will revert after the check finishs. This was added to trigger the checks

Reply:
Reverted